RockLib.Configuration.ObjectFactory/ConstructorOrderInfo.cs

```diff
@@ -22,20 +22,24 @@ public ConstructorOrderInfo(ConstructorInfo constructor, Dictionary<string, ICon
             }
             else
             {
-                bool HasAvailableValue(ParameterInfo p) =>
-                    p.GetNames().Any(name => availableMembers.ContainsKey(name)) || resolver.CanResolve(p);
+                bool HasAvailableValue(ParameterInfo p)
+                    => p.GetNames().Any(name => availableMembers.ContainsKey(name)) || resolver.CanResolve(p);
+                bool HasAvailableNamedValue(ParameterInfo p)
+                    => p.GetNames().Any(name => availableMembers.ContainsKey(name));
```

Comment:
Minor stylistic issue: keep `=>` on the same line as the function name & parameters.

Reply:
Good catch. Fixed.